github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger/help-skipFirstRun.html

```diff
@@ -1,4 +1,7 @@
 <div>
-    If this checked, then first trigger run will not run real builds.
-    After first run this value automatically unset.
+    If selected, PRs that were pending before a first dry run of this job will
+    be ignored, and only PRs updated or created after that first dry run will
+    be built.
+    <br/>
+    NOTE: After selecting this option, you need to run the build once by hand.
```

Comment:
Ok, so let me try to understand - if you're using crons and this is selected, it'll ignore any existing PRs until a new state change happens to one PR (a new one's opened, a comment is added to an existing one, etc), and then this setting will be automatically unselected? Is that right?